src/node/internal/internal_zlib.ts

```diff
@@ -14,6 +14,122 @@ function crc32(data: ArrayBufferView | string, value: number = 0): number {
+BROTLI_DECODER_RESULT_NEEDS_MORE_OUTPUT: zlibUtil.BROTLI_DECODER_RESULT_NEEDS_MORE_OUTPUT,
+BROTLI_DECODER_PARAM_DISABLE_RING_BUFFER_REALLOCATION: zlibUtil.BROTLI_DECODER_PARAM_DISABLE_RING_BUFFER_REALLOCATION,
+BROTLI_DECODER_PARAM_LARGE_WINDOW: zlibUtil.BROTLI_DECODER_PARAM_LARGE_WINDOW,
+BROTLI_DECODER_NO_ERROR: zlibUtil.BROTLI_DECODER_NO_ERROR,
+BROTLI_DECODER_SUCCESS: zlibUtil.BROTLI_DECODER_SUCCESS,
+BROTLI_DECODER_NEEDS_MORE_INPUT: zlibUtil.BROTLI_DECODER_NEEDS_MORE_INPUT,
+BROTLI_DECODER_NEEDS_MORE_OUTPUT: zlibUtil.BROTLI_DECODER_NEEDS_MORE_OUTPUT,
+BROTLI_DECODER_ERROR_FORMAT_EXUBERANT_NIBBLE: zlibUtil.BROTLI_DECODER_ERROR_FORMAT_EXUBERANT_NIBBLE,
+BROTLI_DECODER_ERROR_FORMAT_RESERVED: zlibUtil.BROTLI_DECODER_ERROR_FORMAT_RESERVED,
+BROTLI_DECODER_ERROR_FORMAT_EXUBERANT_META_NIBBLE: zlibUtil.BROTLI_DECODER_ERROR_FORMAT_EXUBERANT_META_NIBBLE,
+BROTLI_DECODER_ERROR_FORMAT_SIMPLE_HUFFMAN_ALPHABET: zlibUtil.BROTLI_DECODER_ERROR_FORMAT_SIMPLE_HUFFMAN_ALPHABET,
+BROTLI_DECODER_ERROR_FORMAT_SIMPLE_HUFFMAN_SAME: zlibUtil.BROTLI_DECODER_ERROR_FORMAT_SIMPLE_HUFFMAN_SAME,
+BROTLI_DECODER_ERROR_FORMAT_CL_SPACE: zlibUtil.BROTLI_DECODER_ERROR_FORMAT_CL_SPACE,
+BROTLI_DECODER_ERROR_FORMAT_HUFFMAN_SPACE: zlibUtil.BROTLI_DECODER_ERROR_FORMAT_HUFFMAN_SPACE,
+BROTLI_DECODER_ERROR_FORMAT_CONTEXT_MAP_REPEAT: zlibUtil.BROTLI_DECODER_ERROR_FORMAT_CONTEXT_MAP_REPEAT,
+BROTLI_DECODER_ERROR_FORMAT_BLOCK_LENGTH_1: zlibUtil.BROTLI_DECODER_ERROR_FORMAT_BLOCK_LENGTH_1,
+BROTLI_DECODER_ERROR_FORMAT_BLOCK_LENGTH_2: zlibUtil.BROTLI_DECODER_ERROR_FORMAT_BLOCK_LENGTH_2,
+BROTLI_DECODER_ERROR_FORMAT_TRANSFORM: zlibUtil.BROTLI_DECODER_ERROR_FORMAT_TRANSFORM,
+BROTLI_DECODER_ERROR_FORMAT_DICTIONARY: zlibUtil.BROTLI_DECODER_ERROR_FORMAT_DICTIONARY,
+BROTLI_DECODER_ERROR_FORMAT_WINDOW_BITS: zlibUtil.BROTLI_DECODER_ERROR_FORMAT_WINDOW_BITS,
+BROTLI_DECODER_ERROR_FORMAT_PADDING_1: zlibUtil.BROTLI_DECODER_ERROR_FORMAT_PADDING_1,
+BROTLI_DECODER_ERROR_FORMAT_PADDING_2: zlibUtil.BROTLI_DECODER_ERROR_FORMAT_PADDING_2,
+BROTLI_DECODER_ERROR_FORMAT_DISTANCE: zlibUtil.BROTLI_DECODER_ERROR_FORMAT_DISTANCE,
+BROTLI_DECODER_ERROR_DICTIONARY_NOT_SET: zlibUtil.BROTLI_DECODER_ERROR_DICTIONARY_NOT_SET,
+BROTLI_DECODER_ERROR_INVALID_ARGUMENTS: zlibUtil.BROTLI_DECODER_ERROR_INVALID_ARGUMENTS,
+BROTLI_DECODER_ERROR_ALLOC_CONTEXT_MODES: zlibUtil.BROTLI_DECODER_ERROR_ALLOC_CONTEXT_MODES,
+BROTLI_DECODER_ERROR_ALLOC_TREE_GROUPS: zlibUtil.BROTLI_DECODER_ERROR_ALLOC_TREE_GROUPS,
+BROTLI_DECODER_ERROR_ALLOC_CONTEXT_MAP: zlibUtil.BROTLI_DECODER_ERROR_ALLOC_CONTEXT_MAP,
+BROTLI_DECODER_ERROR_ALLOC_RING_BUFFER_1: zlibUtil.BROTLI_DECODER_ERROR_ALLOC_RING_BUFFER_1,
+BROTLI_DECODER_ERROR_ALLOC_RING_BUFFER_2: zlibUtil.BROTLI_DECODER_ERROR_ALLOC_RING_BUFFER_2,
+BROTLI_DECODER_ERROR_ALLOC_BLOCK_TYPE_TREES: zlibUtil.BROTLI_DECODER_ERROR_ALLOC_BLOCK_TYPE_TREES,
+BROTLI_DECODER_ERROR_UNREACHABLE: zlibUtil.BROTLI_DECODER_ERROR_UNREACHABLE,
```

Comment:
nit: these aren't actually going to be constant defined this way.  These would likely need to be added use `Object.defineProperties(...)` with `writable: false` and `configurable: false`.